docs/docs/configuration.md

```diff
@@ -281,6 +281,13 @@ The specified file `./secrets/api-key.txt` contains the clear text api key only.
 
 The port mapping will make available the service at `http://localhost:5000`. (To make it available using `https` you should use an additional reverse proxy service, like "apache" or "nginx".)
 
+To enable HTTP protocol on the docker Console you can add the following variable:
+
+```yaml
+environment:
+ - Logging__Console__LogLevel__Microsoft.AspNetCore.HttpLogging.HttpLoggingMiddleware=Information
+```
+
```

Comment:
This is a section specifically about how to load secrets from a file. Please make a new chapter `## Logging`, maybe directly under `##Statistics` just before the secret loading chapter.